### DISCUSSION TITLE
Update documentation for lockFileMaintenance to reflect new manager support

### DISCUSSION THREAD

**Discussion by gresavage** [renovatebot/renovate#32342] [Suggest an Idea]
### Tell us more.

Judging by https://github.com/renovatebot/renovate/issues/27841 and https://github.com/renovatebot/renovate/pull/30756 it appears as though new support for `lockFileMaintenance` has been added in version `38.41.0` but as of `39.0.4` the documentation has not been updated to match.

It's a minor inconvenience - since at first I thought renovatebot lacked support for `uv.lock` but found out this was incorrect after searching through issues.

See below:

![image](https://github.com/user-attachments/assets/e1a6f9c6-6391-4f1c-9356-d09831439d12)

**Comment by rarkins** [comment]
Docs PRs always welcome

**Reply by gresavage** [reply]
I'd be happy to initiate a PR - It might take some time for me to determine the diff between the supported lockfiles in this list and the currently supported ones
